pygmt/src/solar.py

```diff
@@ -0,0 +1,91 @@
+"""
+solar - Plot day-night terminators.
+"""
+
+from pygmt.clib import Session
+from pygmt.helpers import (
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    J="projection",
+    G="fill",
+    B="frame",
+    Td="day_night",
+    Tc="civil_twilight",
+    Tn="nautical_twilight",
+    Ta="astronomical_twilight",
+    W="pen",
+    U="timestamp",
+    V="verbose",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+)
+@kwargs_to_strings(R="sequence", p="sequence")
+def solar(self, **kwargs):
+    r"""
+    A function that plots the day-night terminator. Additionally, it can
+    plot the terminators for civil twilight, nautical twilight, and
+    astronomical twilight.
```

Comment:
The first sentence in docstring is special and shown as the summary of the function. 
```suggestion
    r"""
    Plot day-light terminators.
    
    This function plots the day-night terminator. Additionally, it can
    plot the terminators for civil twilight, nautical twilight, and
    astronomical twilight.
```